megamek/src/megamek/client/ui/swing/ClientGUI.java

```diff
@@ -202,7 +248,7 @@
     /**
      * Map each phase to the name of the card for the main display area.
      */
-    private HashMap<String, String> mainNames = new HashMap<String, String>();
+    private HashMap<String, String> mainNames = new HashMap<>();
```

Comment:
Might as well go through and make these Maps if possible.

Reply:
Fixed